animatedModal.js

```diff
@@ -9,6 +9,7 @@
 (function ($) {
  
     $.fn.animatedModal = function(options) {
+        var me = this;
```

Comment:
holding variable so we can return "this" at the end as is customary (with added openModal and closeModal methods).